packages/app-project/components/ClassifyPage/components/ClassifierWrapper/components/ErrorMessage/ErrorMessage.js

```diff
@@ -0,0 +1,55 @@
+import counterpart from 'counterpart'
+import { Box, Heading, Text } from 'grommet'
+import { shape, string } from 'prop-types'
+import React from 'react'
+import styled from 'styled-components'
+
+import en from './locales/en'
+
+counterpart.registerTranslations('en', en)
+
+const Pre = styled.pre`
+  margin: 0;
+`
+
+export default function ErrorMessage ({ error }) {
+  return (
+    <Box
+      background='#FFF2E9'
+      border={{ color: 'tomato' }}
+      pad='medium'
+    >
+      <Heading
+        level='3'
+        margin='none'
+        size='small'
+      >
+        {counterpart('ErrorMessage.title')}
+      </Heading>
+
+      {error.stack && (
+        <Text size='small' margin={{ top: 'medium' }}>
+          <Pre>
+            {error.stack}
+          </Pre>
+        </Text>
+      )}
+
+      {!error.stack && (
```

Comment:
If there's no stack trace, don't we still want to know what the error was?

Reply:
I think the message should be printed before the stack trace too. See the screenshot in zooniverse/Panoptes-Front-End#5235

Reply:
If there's a stack trace, the error name and message is printed as part of that. If there isn't, you get the name and message as per line 41

Reply:
That's useful. `componentDidCatch` only gives you the stack.